sdk/storage/azure-storage-file/src/test/java/com/azure/storage/file/DirectoryAPITests.groovy

```diff
@@ -90,18 +90,6 @@ class DirectoryAPITests extends APISpec {
         FileTestHelper.assertResponseStatusCode(primaryDirectoryClient.createWithResponse(null, null, testMetadata, null, null), 201)
     }
 
-    def "Create directory error with metadata"() {
```

Comment:
Is this a duplicate test?

Reply:
It is there to test the metadata with capital key.
Since we accept the case, it is no longer an error case.

Reply:
It might be best to call this PR Fix file and queue failure tests after string to sign token changes